src/pudl/transform/eia861.py

```diff
@@ -661,6 +679,84 @@ def _compare_totals(data_cols, idx_cols, class_type, df_name):
                 f'{df_name}: for column {col} all total values are NaN')
 
 
+def _clean_nerc_add_rows(df, idx_cols, idx_no_nerc):
+    """Clean NERC region entries and make new rows for multiple nercs.
+
+    This function examines reported NERC regions and makes sure the output column of
+    the same name has reliable, singular NERC region acronyms. To do so, this function
+    identifies entries where there are two or more NERC regions specified in a single cell
+    (such as SPP & ERCOT) and makes new, duplicate rows for each NERC region. It also
+    converts non-recognized reported nerc regions to 'UNK'.
+
+    Args:
+        df (pandas.DataFrame): A DataFrame with the column 'nerc_region' to be
+            cleaned.
+        idx_cols (list): A list of the primary keys.
+        idx_no_nerc (list): A list of the primary keys, not including nerc_region.
+
+    Returns:
+        pandas.DataFrame: A DataFrame with correct and clean nerc regions.
+
+    """
+    # Split raw df into primary keys plus nerc region and other value cols
+    nerc_df = df[idx_cols]
+    other_df = df.drop('nerc_region', axis=1).set_index(idx_no_nerc)
+
+    # Make all values upper-case
+    # Replace all NA values with UNK
+    # Make nerc values into lists to see how many separate values are stuffed into one row (ex: SPP & ERCOT)
+    nerc_df = (
+        nerc_df.assign(
+            nerc_region=(lambda x: (
+                x.nerc_region
+                .str.upper()
+                .fillna('UNK')
+                .str.findall(r'[A-Z]+'))),
+            multiple_nercs=(lambda x: (
+                x.nerc_region.apply(lambda x: len(x) > 1)))
```

Comment:
This one was complicated. In this case, x is actually actually temporary list within the column cell. The function reads the moosh of data in the `nerc_region` column: if it's something like `SPP` then it becomes `['SPP']` but if it's something like `SPP & ERCOT` it becomes `['SPP', 'ERCOT']`. The `len(x) > 1` is looking for instances where there are more than one NERC region in a given row (i.e., the list is greater than 1) and then splits those into multiple rows so that there is only one nerc region per row. The single nerc values are then turned to strings and validated against recognized nerc regions. If you can think of a more efficient way to do this I would love to know! Right now it needs to be a list rather than a boolean so I can extract the region name later when the rows are separated. I'll keep poking at it.

Reply:
But, looking at it again, I think your suggestion might still work? I'll have to take a closer look.

Reply:
do not feel obligated to change this, because it definitely looks like it works as is!

Reply:
It looks like the function still needs an apply to reach the actual cell value as opposed to the series. I was able to change it to `df.nerc_region.apply(lambda x: np.where((len(x) > 1), True, False))` but I'm not sure that makes a big difference.